### ISSUE TITLE
Graphical difference between directories and pictures

### ISSUE THREAD

**Issue by JulienPalard** [saimn/sigal#190]
Hi,

I think sigal may be friendlier if one can distinct between a picture and a directory. Actual themes provide identical view for a pic thumbnail and a directory thumbnail, and I'm almost stressed while clicking on a thumbnail as I don't know how sigal will react "will it open a directory of pop this pic in fullscreen" :-p

I typically like the solution used by gwenview of displaying pictures side by side like they were dropped:

![gwenview thumbnails](http://1.bp.blogspot.com/-chjsd7Hrl5s/UT75AwOeDrI/AAAAAAAAGWw/tpqnK8xkA_4/s1600/gwenview.jpg)

**Comment by saimn** [comment]
> "will it open a directory of pop this pic in fullscreen" :-p

:-)
You're right, but I don't know how to show the difference, and I don't like much the folder with multiple images. Will think about this, other suggestions are welcome.

**Comment by JulienPalard** [comment]
> I don't like much the folder with multiple images

Makes me sad.

Is that because displaying them side-by-side makes them too little ? If it's the case, we may just stack them in a single stack, at a near full size, with only a random in the angle and a near-to-zero random in the position, just to show there's multiple image, but readlly displaying a single one. (Also, as only a few will be visible, we may only render it using like five images). Following the mother nature idea:

![mother nature directory of pictures](http://www.publicdomainpictures.net/pictures/30000/nahled/photo-stack.jpg)

May look like:

![image stack](http://ui-cloud.com/res/pixelsdaily/CSS_Image_Stacks/previews/css-image-stacks1.png)

(I only searched "stack of pictures" on google image)

**Comment by saimn** [comment]
Yep, this would be nice, a simple effect like http://www.webappers.com/2012/06/04/create-stacked-images-with-css3-pseudo-elements/

**Comment by JulienPalard** [comment]
You're right, no need to actually see the content of the other pictures, just the white border is enough, and that's a lot easier to produce (no need to access actual files). It would be OK to me.

**Comment by saimn** [comment]
I tried something like this but I'm not convinced by the result:
![2016-03-20-223201_801x547_scrot](https://cloud.githubusercontent.com/assets/311639/13907327/febc7f30-eeeb-11e5-832e-89ddb68d60cc.png)

The code is hre : https://github.com/saimn/sigal/commit/66fda76e517b3ecacda65c02a3f60caf4d270c4d (in case you or someone else wants to experiment ;))

**Comment by JulienPalard** [comment]
In case someone else wants to experiment, there's also very … strangly experimental hipster-nerdy-psychedelic library for stacked effects : https://tympanus.net/Development/StackMotionHoverEffects/

**Comment by amittel** [comment]
Hi!
Discovered sigal while sorting my photos from building my bike back up.
If anyone is interested I could upload the theme. But it's not finished.
I used [Semantic UI](https://semantic-ui.com) to create it.
![semanticui](https://cloud.githubusercontent.com/assets/822396/26389846/36ee11bc-405c-11e7-8df3-76b39fcb6e62.png)

**Comment by saimn** [comment]
@amittel - Nice, it always good to have more choice for the themes so it would be good to have it.

**Comment by saimn** [comment]
Another idea (from [lychee](https://ld.electerious.com/) and probably others): put an overlay on the bottom of the thumbnail with the album title and number of images

![capture d ecran de 2018-03-27 23-26-10](https://user-images.githubusercontent.com/311639/37995945-562c7558-3216-11e8-9f70-2a3920cbd3ec.png)

**Comment by DuendeInexistente** [comment]
I came here for this, we need at least some css class to style folders differently from files.
